docs/blogs/2023/2023-06-07-fast-conformer.md

```diff
@@ -0,0 +1,210 @@
+| Conformer           	| 31.0 	|    267    	|    1X    	|
+| Fast Conformer      	| 31.4 	|    161    	|   1.66X  	|
+|     RNNT decoder    	|      	|           	|          	|
+| Conformer           	| 23.2 	|     83    	|    1X    	|
+| Fast Conformer      	| 27.9 	|     45    	|   1.84X  	|
+
+## Spoken Language Understanding Results
+
+For the Speech Intent Clasification and Slot Filling task, experiments are conducted using the [SLURP](https://arxiv.org/abs/2011.13205) dataset, where intent accuracy and SLURP-F1 are used as the evaluation metric. For a fair comparison, both Conformer and Fast Conformer models are initialized by training on the same NeMo ASR Set dataset (roughly 25,000 hours of speech) and then the weights of the entire model are finetuned with the respective decoders.
+
+*Speech intent classification and slot filling on SLURP dataset. ESPNet-SLU  and SpeechBrain-SLU models use a [HuBERT](https://arxiv.org/abs/2106.07447) encoder pre-trained via a self-supervised objective on [LibriLight-60k](https://arxiv.org/abs/1912.07875). 
+Inference time and relative speed-up against Conformer are measured with batch size 32.*
+
+| Model                                        	| Intent Acc 	| SLURP F1 	| Inference, sec 	| Rel. Speed-up 	|
+|----------------------------------------------	|:----------:	|:--------:	|:--------------:	|:-------------:	|
+| SpeechBrain-SLU                              	|    87.70   	|   76.19  	|        -       	|       -       	|
+| ESPnet-SLU                                   	|    86.52   	|   76.91  	|        -       	|       -       	|
+| Conformer/Fast Conformer+Transformer Decoder 	|            	|          	|                	|               	|
+| Conformer                                    	|    90.14   	|   82.27  	|       210      	|       1X      	|
+| Fast Conformer                               	|    90.68   	|   82.04  	|       191      	|      1.1X     	|
+
+## Long Form Speech Recognition Results
+
+While Fast Conformer can be modified post training to do simple inference on long audio, due to the mismatch in attention window with limited future information, the model's WER may degrade a small amount. Users can therefore add global token followed by subsequent fine-tuning for some small steps on the same dataset in order to significantly recover (and outperform) the original models WER.
+
+Note that with Longformer style attention, we can still perform buffered inference with large chunk size - upto 1 hour long, therefore inference on even longer audio can be done efficiently with few inference steps.
+
+*Fast Conformer versus Conformer on long audio. We evaluated four versions of FastConformer: (1) FC with global attention (2) FC with limited context (3) FC with limited context and global token (4) FC with limited context and global token, trained on long concatenated utterances. Models have been evaluated on two long-audio bencmarks: TED-LIUM v3 and Earning 21. Greedy WER(%).*
+
```

Comment:
is it possible to center the table to the page?

Reply:
Tried to use div align center but it ruins the markdown table generation. Lets skip for now.